libs/agno/agno/workflow/v2/types.py

```diff
@@ -185,6 +185,9 @@ class StepOutput:
     videos: Optional[List[VideoArtifact]] = None
     audio: Optional[List[AudioArtifact]] = None
 
+    # Metrics for this step execution
+    metrics: Optional[Dict[str, Any]] = None
```

Comment:
Also a type here pls

Reply:
But these are the metrics directly from the agent/team, if any thing changes from there the type will break...
in teams/agent also we have the `Optional[Dict[str, Any]]` type. i think we should put the type just on the main workflow level metrics.